macros/Cargo.toml

```diff
@@ -1,7 +1,18 @@
 [package]
-name = "uuid_macro"
-version = "0.0.0"
+name = "uuid-macro-internal"
+version = "1.0.0-alpha.1"
 edition = "2018"
+authors = [
+    "QnnOkabayashi"
```

Comment:
Should this include atleast one `uuid-rs` team member?

Unless I misunderstand how initial ownership for crates.io works

Reply:
This field’s just an acknowledgement of the primary authors so I thought I’d put just @QnnOkabayashi here for now as the current sole contributor to this piece.

Reply:
ah yes.. ok. 

clearly I don't understand basic ownership. 😅

Reply:
Ah yes, it's separate from the ownership rights on crates.io 🙂